dts: i2c: Simplify the description of the binding

rruuaanng opened the pull request:
Remove redundant descriptions in I2C bindings, such as "This is a representation of" and "... node".

kartben commented on dts/bindings/i2c/andestech,atciic100.yaml:
```diff
@@ -4,8 +4,7 @@
 
-description:
-  This is a representation of AndesTech atciic100 I2C node
+description: AndesTech atciic100 I2C node
```
```suggestion
description: AndesTech ATCIIC100 I2C
```

kartben commented on dts/bindings/i2c/nuvoton,npcx-i2c-port.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX-I2C port pads node
+description: Nuvoton, NPCX-I2C port pads
```
```suggestion
description: Nuvoton NPCX-I2C port pads
```

kartben commented on dts/bindings/i2c/nuvoton,npcx-i2c-ctrl.yaml:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX-I2C controller node
+description: Nuvoton, NPCX-I2C controller
```
```suggestion
description: Nuvoton NPCX-I2C controller
```

rruuaanng committed:
dts: i2c: Simplify the description of the binding

Remove redundant descriptions in I2C bindings, such
as "This is a representation of" and "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>
